frontend/src/metabase/querying/filters/components/DatePicker/SpecificDatePicker/DateRangePicker/DateRangePickerBody/DateRangePickerBody.tsx

```diff
@@ -20,12 +20,14 @@ interface DateRangePickerBodyProps {
   value: [Date, Date];
   hasTime: boolean;
   onChange: (value: [Date, Date]) => void;
+  valueFormat?: string;
 }
 
 export function DateRangePickerBody({
   value,
   hasTime,
   onChange,
+  valueFormat = "LL",
```

Comment:
@egorgrushin to confirm - that's not set anywhere, correct? perhaps we want a constant instead. It would be also cool to have a unit test that verifies the change